opendevin/config.py

```diff
@@ -75,8 +75,10 @@ def get(key: str):
     return config.get(key)
 
 
-def get_all() -> dict:
+def get_fe_config() -> dict:
     """
     Get all the configuration values by performing a deep copy.
     """
-    return copy.deepcopy(config)
+    fe_config = copy.deepcopy(config)
+    fe_config['LLM_API_KEY'] = '0000000'
```

Comment:
what's this about?

Reply:
```suggestion
    del fe_config['LLM_API_KEY']
```

Reply:
I think I tested with nothing and with ***, and some React object died, then AgentController wouldn't initialize. Maybe it was a fluke though, lets try.